src/app/ApplicationTemplate.Access/Configuration/SerializationConfiguration.cs

```diff
@@ -15,7 +14,7 @@ namespace ApplicationTemplate;
 [JsonSerializable(typeof(AuthenticationToken))]
 [JsonSerializable(typeof(AuthenticationData))]
 [JsonSerializable(typeof(ApplicationSettings))]
-[JsonSerializable(typeof(Refit.ProblemDetails))]
```

Comment:
Not sure I would change this. Without the prefix it looks like ProblemDetails comes from the application and not from the Refit Library.

Reply:
Sure.